atlasdb-service-server/src/main/java/com/palantir/atlasdb/server/AtlasDbServiceServerConfiguration.java

```diff
@@ -25,7 +25,7 @@
 
     private final AtlasDbConfig atlasdb;
 
-    public AtlasDbServiceServerConfiguration(@JsonProperty(AtlasDbConfigs.ATLASDB_CONFIG_ROOT) AtlasDbConfig atlasdb) {
+    public AtlasDbServiceServerConfiguration(@JsonProperty(AtlasDbConfigs.ATLASDB_CONFIG_OBJECT_NAME) AtlasDbConfig atlasdb) {
```

Comment:
also if we rename the service to "timelock" or something we should re-hardcode it to that.  Unless this code is already in use somewhere?